pyproject.toml

```diff
@@ -5,7 +5,6 @@
 
 [tool.black]
 line-length = 120
-py36 = false
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Mostly for internal doc, why was this option removed ?

Reply:
Running black with this option prints a warning saying this option is unused. It seems it was deprecated in 2019 (https://github.com/psf/black/pull/724). From what I've been reading, black does automatic detection based on language features if the minimum Python version is not specified, so there's no need to disable explicitly python3.6.